L.TileLayer.PouchDBCached.js

```diff
@@ -177,6 +178,20 @@ L.TileLayer.include({
 		this._seedOneTile(tile, queue, seedData);
 	},
 
+	_createTile: function () {
+		return document.createElement('img');
+	},
+
+	_getTileUrl: function (coords) {
+		return L.Util.template(this._url, L.extend({
+			r: this.options.detectRetina && L.Browser.retina && this.options.maxZoom > 0 ? '@2x' : '',
+			s: this._getSubdomain(coords),
+			x: coords.x,
+			y: this.options.tms ? this._globalTileRange.max.y - coords.y : coords.y,
+			z: coords.z
```

Comment:
It should be a good idea to add a (short) comment about how `_getTileUrl()` is different from `getTileUrl()` and why it's needed.